### DISCUSSION TITLE
What is `qpos` actually? (A suggestion to improve the documentation)

### DISCUSSION THREAD

**Discussion by Kallinteris-Andreas** [google-deepmind/mujoco#2016] [Asking for Help]
Hi,

I'm a maintainer of [Gymnasium](https://gymnasium.farama.org/) & the project manager of [Gymnasium-Robotics](https://robotics.farama.org/), and I'm trying to use `MuJoCo` for "

`qpos` is the generalized coordinates of joints  (according to https://mujoco.readthedocs.io/en/stable/computation/index.html#physics-state)

But what "coordinates" exactly?
It appears to be relative to the original position.

But for free joints It is the absolute coordinates relative to (0, 0, 0) (which is stated in https://mujoco.readthedocs.io/en/stable/overview.html#floating-objects)

**Comment by yuvaltassa** [comment]
Do you want to make your suggestion in the form of a PR?
